map/framework.cpp

```diff
@@ -1339,6 +1342,8 @@ bool Framework::SearchEverywhere(search::EverywhereSearchParams const & params)
   p.m_query = params.m_query;
   p.m_inputLocale = params.m_inputLocale;
   p.m_mode = search::Mode::Everywhere;
+  if (m_isViewportInitialized)
```

Comment:
I cannot see these checks in the old code. Could you please explain why they are important?

Reply:
We've discussed this offline.
Completely filled params are expected by the time they will be used, but no other guarantees are given. So all fields of params are filled here except for viewport, which is filled later for delayed intents.
The old code behaved the same, overwriting the previously written viewport with the actual one for the delayed intents.

It's not clear whether we can fill all params in one place but it doesn't seem that we can.